src/geocat/f2py/triple_to_grid_wrapper.py

```diff
@@ -466,18 +472,17 @@ def triple_to_grid(
     # If input was xarray.DataArray, convert output to xarray.DataArray as well
     if is_input_xr:
         grid = xr.DataArray(grid)
+    # Else if input was numpy.ndarray, convert Dask output to numpy.ndarray with `.compute()
+    else:
+        grid = grid.compute()
 
     return grid
 
 
 # TODO: Revisit for implementing this function after deprecating geocat.ncomp
 def triple_to_grid_2d(x_in, y_in, data, x_out, y_out, msg_py):
-    warnings.warn(
-        "triple_to_grid_2d function name and signature will be deprecated soon "
-        "in a future version. Use `grid_to_triple` instead!",
-        PendingDeprecationWarning)
 
-    return triple_to_grid(data, x_in, y_in, x_out, y_out, missing_value=msg_py)
+    warnings.warn("triple_to_grid_2d function not yet implemented!!! ")
```

Comment:
triple_to_grid exists, this function should call it, and continue to throw a pending deprecation warning about the name triple_to_grid_2d.

Reply:
`triple2grid` and `triple2grid2d` were two separate functions in NCL whose return grid types and dimensionality were different. So, `triple_to_grid_2d` here refers to NCL's `triple2grid2d` and not `triple2grid`. That's why  the warning here is not a deprecation warning, instead it is a "not yet implemented but the signature is here" warning.

Reply:
OK, leave as is then. Resolved.